src/main/scala/org/ergoplatform/nodeView/wallet/WalletCache.scala

```diff
@@ -57,9 +57,8 @@ object WalletCache {
     * Constructing a Bloom filter for scanning the boxes efficiently
     */
   def filter(trackedBytes: Seq[Array[Byte]],
```

Comment:
Consider a more descriptive name, like for example `createFilterFor`

Reply:
changed to createFilter